SingularityService/src/main/java/com/hubspot/singularity/data/SingularityValidator.java

```diff
@@ -32,6 +33,7 @@
 import com.hubspot.singularity.WebExceptions;
 import com.hubspot.singularity.config.SingularityConfiguration;
 import com.hubspot.singularity.data.history.DeployHistoryHelper;
+import org.dmfs.rfc5545.recur.RecurrenceRule;
```

Comment:
Wow that re-formatted everything, perhaps I should just re-format the sections I added?